submissions/api.py

```diff
@@ -467,8 +469,8 @@ def get_top_submissions(course_id, item_id, item_type, number_of_top_scores, use
             for score in scores
         ]
 
-    # Always store the retrieved list in the cache
-    cache.set(cache_key, top_submissions, TOP_SUBMISSIONS_CACHE_TIMEOUT)
+        # Always store the retrieved list in the cache
+        cache.set(cache_key, top_submissions, TOP_SUBMISSIONS_CACHE_TIMEOUT)
```

Comment:
@stephensanchez does this make sense to set the cache only after we hit the database. Please let me know what do you think ? 
Thanks.